<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ocean</class>
 <widget class="QMainWindow" name="Ocean">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ocean</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="main_layout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="OceanScene"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="ConfigLaout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="okeanNameLabel">
          <property name="font">
           <font>
            <family>AR JULIAN</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>O c e a n</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="topMargin">
           <number>19</number>
          </property>
          <property name="horizontalSpacing">
           <number>78</number>
          </property>
          <item row="1" column="0">
           <widget class="QLabel" name="SizeLabel">
            <property name="text">
             <string>Размер поля</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="SizeFieldBox">
            <property name="minimum">
             <number>30</number>
            </property>
            <property name="maximum">
             <number>400</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="UpTimeBox">
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>5000</number>
            </property>
            <property name="value">
             <number>150</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="UpTimeLabel">
            <property name="text">
             <string>Скорость (мкс)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="topMargin">
           <number>22</number>
          </property>
          <item row="1" column="2">
           <widget class="QLabel" name="LabelVictims">
            <property name="text">
             <string>Жертва</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="Amount_Predators">
            <property name="maximum">
             <number>5000</number>
            </property>
            <property name="value">
             <number>25</number>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QSpinBox" name="Amount_Victims">
            <property name="maximum">
             <number>5000</number>
            </property>
            <property name="value">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Хищник</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QSpinBox" name="razmnozh_Victims">
            <property name="value">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QSpinBox" name="timeLife_Victims"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="LabelPredators">
            <property name="text">
             <string>Начальное количество</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="timeLife_Predators"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Размножение (кол-во ходов)</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Продолжительность жизни </string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="razmnozh_Predators"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Гибель голодной клетки</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QSpinBox" name="deadStep_Predators">
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="topMargin">
           <number>25</number>
          </property>
          <item row="0" column="0">
           <widget class="QPushButton" name="startButton">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(211, 214, 255);
    border: 2px groove gray;
    border-radius: 10px;
    padding: 2px 4px;</string>
            </property>
            <property name="text">
             <string>Старт</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="stopButton">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(211, 214, 255);
    border: 2px groove gray;
    border-radius: 10px;
    padding: 2px 4px;background-color: rgb(211, 214, 255);</string>
            </property>
            <property name="text">
             <string>Стоп</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="clearButton">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(211, 214, 255);
    border: 2px groove gray;
    border-radius: 10px;
    padding: 2px 4px;</string>
            </property>
            <property name="text">
             <string>Очистить</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButton">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(211, 214, 255);
    border: 2px groove gray;
    border-radius: 10px;
    padding: 2px 4px;</string>
            </property>
            <property name="text">
             <string>Генерация поля</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_6">
          <property name="topMargin">
           <number>27</number>
          </property>
          <item row="3" column="1">
           <widget class="QLabel" name="countVictims">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="countPredators">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Количество жертв:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Количество хищников:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Статистика:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_5"/>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
